app/src/main/java/Player.java

```diff
@@ -8,13 +8,15 @@ public class Player implements java.io.Serializable{
     private String name;
     private String position;
     private int number;
+    private boolean isPlaying;
 
     @JsonCreator
     public Player(@JsonProperty("name") String name, @JsonProperty("position") String position,
                   @JsonProperty("number") int number) {
```

Comment:
I think needs an at property so that the Player json is sent over with the an isPlaying field